### PR TITLE
Getting real data via new APIs and adding 2 new pages

### PR DESCRIPTION
**Pull request by Laura-TM**
Screenshot 2021-07-21 at 18 39 24
Screenshot 2021-07-21 at 18 39 46
Screenshot 2021-07-21 at 18 40 09

Still contains hardcoded list of users and channel information

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -11,6 +13,12 @@ const App = () => (
 		<Route path="/about/this/site">
 			<About />
 		</Route>
```
Comment (askinekincihf) [inline client/src/App.js:15]: Hi Laura, do we have to still keep the About page, if so, what is your opinion to fill in?

**Review by askinekincihf — COMMENTED** [review]
Well done Laura, it is great to have a good-looking front-end.  I like the colour selections. I have got a couple of questions because I haven't used 'reactstrap'.
1. Have you done all pages responsive?
2. Can we do logos on the channel page a bit smaller to focus on the data more?
3. Can we also add a footer with social media links ( if we have a time, we can add later.)

One you merge the front end, we will have a discussion how to make a comparison on the data to give some notification to the mentors.

File: client/src/App.js (hunk 1 above)
Comment (Laura-TM) [reply client/src/App.js:15]: Not sure if we need the About page, as I have added an intro on the Home page that explains the overall concept; but let's keep the route for now and maybe we can add it to the Footer.

**Comment by Laura-TM** [comment]
> Well done Laura, it is great to have a good-looking front-end. I like the colour selections. I have got a couple of questions because I haven't used 'reactstrap'.
> 
> 1. Have you done all pages responsive?
> 2. Can we do logos on the channel page a bit smaller to focus on the data more?
> 3. Can we also add a footer with social media links ( if we have a time, we can add later.)
> 
> One you merge the front end, we will have a discussion how to make a comparison on the data to give some notification to the mentors.

1. Not yet, I will add media queries (but this is not related to reactstrap)
2. Yes, good idea (I will commit this as soon as possible)
3. Yes, certainly, it will be added to the next PR

**Comment by askinekincihf** [comment]
Good to see the changes, thanks Laura.

**Comment by DibaKarimi** [comment]
Hi Laura, I put comment to check can you merge it or not

**Pull request merged**